src/main/kotlin/sliv/tool/scene/model/Utils.kt

```diff
@@ -0,0 +1,13 @@
+package sliv.tool.scene.model
+
+import javafx.scene.paint.Color
+import kotlin.random.Random
+
+fun generateRandomColor(): Color {
+    val red = generateRandomIntFrom0To255()
+    val green = generateRandomIntFrom0To255()
+    val blue = generateRandomIntFrom0To255()
+    return Color.rgb(red, green, blue)
+}
+
+private fun generateRandomIntFrom0To255() = ((Random.nextInt() % 255) + 255) % 255
```

Comment:
is there any function that takes range and geenarte int from default library?

Reply:
also double mod is stange

Reply:
Found the func from the default lib. What is double mod? There is no Double values

Reply:
it means that you apply mod operator twice